Nef_S2/include/CGAL/Nef_S2/Sphere_segment.h

```diff
@@ -37,14 +37,15 @@ Sphere_segment_rep() { ps_ = pt_ = Point(); c_ = Circle(); }
 
 Sphere_segment_rep(const Point& p1, const Point& p2,
                    bool shorter_arc=true) :
-  ps_(p1), pt_(p2), c_(Plane_3(p1,p2,Point_3(CGAL::ORIGIN)))
+  ps_(p1), pt_(p2),
+  c_(CGAL::ORIGIN,R_().construct_orthogonal_vector_3_object()(p1,p2,Point_3(CGAL::ORIGIN)))
```

Comment:
Is it correct that the three parameters could be in any order?

Reply:
I don't think so. If p1 and p2 have reversed the vector points in the opposite direction? 
Either way, I didn't change the order.